backend/src/directors/report_director.py

```diff
@@ -0,0 +1,23 @@
+
+from typing import TypedDict
+from fastapi import UploadFile
+
+from src.utils.scratchpad import clear_scratchpad, update_scratchpad
+from src.utils.file_utils import handle_file_upload
+
+class FileUploadReport(TypedDict):
+    id: str
+    filename: str | None
+    report: str | None
+
+async def report_on_file_upload(upload:UploadFile) -> FileUploadReport:
+
+    file = handle_file_upload(upload)
+
+    update_scratchpad(result=file["content"])
+
+    report = "#Report on upload as markdown" # await report_agent.invoke(file["content"])
```

Comment:
are we supposed to have this bit of commented out code?
and i this report just a placeholder or something? what is it used for

Reply:
I believe this is the part where we will plug in the report agent once it's complete. For now, it just returns some placeholder markdown

Reply:
Yes, this is just a sample response and to show where the report agent will be integrated. The placeholder report isn't used by anything yet.